nostalgia/sources/ns_chipkaart.py

```diff
@@ -0,0 +1,15 @@
+import just
+import pandas as pd
+
+from nostalgia.ndf import NDF
+from nostalgia.times import parse_date_tz
+
+
+class NsChipkaart(NDF):
+    @classmethod
+    def load(cls, files = "~/nostalgia_data/input/ns_chipkaart/reistransacties-*.xls",  nrows=None):
```

Comment:
The glob should be moved outside of the arguments, just as a constant on the line below :)

Reply:
Yep, will work on it